src/main/java/com/example/demo/security/authInfo/AuthResponse.java

```diff
@@ -9,4 +9,6 @@
 public class AuthResponse { // POJO
     private String username;
     private String accessToken;
+
+    //todo add accountId field so that only orderbook of authorized account can be retrieved
```

Comment:
Is this something you need to make the tests pass?

Reply:
It is one of the things. I'm going to change the system so that either we always use the username to refer to a user, or always use the accountId. In fact, now I'm typing this, I may as well just assume usernames will be unique and use that to uniquely identify each account...